src/fklearn/training/regression.py

```diff
@@ -186,6 +186,111 @@ def p(new_df: pd.DataFrame, apply_shap: bool = False) -> pd.DataFrame:
 xgb_regression_learner.__doc__ += learner_return_docstring("XGboost Regressor")
 
 
+@curry
+@log_learner_time(learner_name='cat_boost_regressor_learner')
+def CatBoostRegressor_learner(df: pd.DataFrame,
```

Comment:
Functions names in python are usually lowercase. Can you please change this to `catboost_regressor_learner` before we merge? Sorry about the delay on detecting this, should have pointed that on my first review.

Reply:
Done.